File: C#/URI/1114.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Globalization;

class URI
{

    static void Main(string[] args)
    {
       int senha = 0000;

        while  (senha != 2002)
        {
            senha = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine(senha == 2002 ? "Acesso Permitido" : "Senha Invalida");

        }

    }
}

File: C/URI/1114.c
#include <stdio.h>
 
int main() {
 
  int senha;
        while(senha != 2002)
        {
                scanf("%d", &senha);
        if(senha == 2002)
                printf("Acesso Permitido\n");
        else
                printf("Senha Invalida\n");
        }


 
    return 0;
}